src/cts/src/TritonCTS.cpp

```diff
@@ -516,7 +519,7 @@ void TritonCTS::inferBufferList(std::vector<std::string>& buffers)
     if (buffers.empty()) {
       logger_->error(
           CTS,
-          110,
+          117,
```

Comment:
Rather than change an existing message you should change the new message

Reply:
This is a new message.  The message ID got changed due to an overlap.

Reply:
No
```
      logger_->error(
          CTS, 110, "Physical master could not be found for cell '{}'", name);
```
is the new message and its id should be changed if there is a conflict.

Reply:
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/761514/034f3921-9a61-4e73-ad62-d11adc046123)
 shows the preceding lines are unmodified so it can't be a new message in this PR

Reply:
Swapped two message IDs

Reply:
did you push the change?

Reply:
Sorry, forgot to add "ready to sync public" label.  Done now.